docs/topics/auth/customizing.txt

```diff
@@ -746,20 +746,17 @@ providing two additional methods:
                 # create user here
                 ...
 
-    .. method:: models.CustomUserManager.create_superuser(*username_field*, password, **other_fields)
+    .. method:: models.CustomUserManager.create_superuser(*username_field*, password=None, **other_fields)
```

Comment:
This is not related with this PR but we should remove asterisks around `username_field` (here and in other methods in this doc).

Reply:
41b3ebb021806ee269f4ca75a28dcbad114ec209